Update the tools to work with the Kyma subscription version v1alpha1 (#2)
kyma-project/eventing-tools: docs/loadtest/README.md

# Load test

Use this tool when JetStream is used as the active Eventing backend.

> The load test tool generates continuous load on the Eventing components by sending CloudEvents to the Eventing publisher proxy forever, and consuming them inside Kyma functions

Reviewer: ```suggestion
The load test tool generates continuous load on the Eventing components by sending CloudEvents to the Eventing publisher proxy forever, and consuming them inside Kyma functions.
```

# Loadtest

> This tool is used to generate continues load on the Eventing components.

Reviewer: just guessing

```suggestion
This tool is used to generate continuous load on the Eventing components.
```

> # Loadtest

Reviewer: I missed this during the first review.
You can always feel free to apply suggested changes to occurrences that I missed because they appear elsewhere in the document.

```suggestion
# Load test
```

> This tool is used to generate continuous load on the Eventing components.

Reviewer: ```suggestion
Use this tool when JetStream is used as the active Eventing backend.

The load test tool generates continuous load on the Eventing components by sending CloudEvents to the Eventing publisher proxy forever, and consuming them inside Kyma functions
```

> It does that by sending CloudEvents to the Eventing publisher proxy forever and consume them inside Kyma functions.

Reviewer: ```suggestion
```

> Ideally, it should be used when JetStream is used as the active Eventing backend.

Reviewer: We can phrase more assertively. Also, we can put this right at the beginning, because then users who don't use JetStream can stop reading right away.

```suggestion
```

> ## Eventing infrastructure

Reviewer: In line 9, it should be **CloudEvents**, not **Cloud Events**

Author: Done

2. **Events carry the `eps` information**:
 - EPS **50**: `order.created.v0050`.
 - EPS **90**: `order.created.v0090`.

> *Cloud Events* have the `eps` encoded at the end as part of the `version`.

Reviewer: ```suggestion
*CloudEvents* have the `eps` encoded at the end as part of the `version`.
```

2. **Events carry the `eps` information**:
 - EPS **50**: `order.created.v0050`.
 - EPS **90**: `order.created.v0090`.

*CloudEvents* have the `eps` encoded at the end as part of the `version`.
*Legacy Events* have the `eps`  body in the `event-type` field of the body.

> > Note: Encoding the EPS in the event type version is used only for debugging purposes and is not a production use-case.

Reviewer: ```suggestion
> Note: Encoding the EPS in the event type version is used only for debugging purposes and is not a production use case.
```

Command-line arguments:

| Arguments | Description                                                                        | Default  |
|-----------|------------------------------------------------------------------------------------|----------|

> | addr      | The loadtest server address used by the Kubernetes liveness and readiness probes.  | :8888    |

Reviewer: ```suggestion
| addr      | The load test server address used by the Kubernetes liveness and readiness probes.  | :8888    |
```
"load test" as a normal English compound noun has a blank space

> [ConfigMap](../../resources/loadtest/300-configmap.yaml) to change the loadtest behaviour at runtime:

Reviewer: ```suggestion
[ConfigMap](../../resources/loadtest/300-configmap.yaml) to change the load test behaviour at runtime:
```

| Config                  | Description                                                                                                             | Default                                             |
|-------------------------|-------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|

> | publish_endpoint        | The Eventing publisher proxy cloudevents endpoint.                                                                      | http://eventing-publisher-proxy.kyma-system/publish |

Reviewer: ```suggestion
| publish_endpoint        | The Eventing publisher proxy CloudEvents endpoint.                                                                      | http://eventing-publisher-proxy.kyma-system/publish |
```
